ui/app/AppLayouts/Wallet/panels/DerivedAddressesPanel.qml

```diff
@@ -117,6 +138,18 @@ Item {
             enabled: RootStore.derivedAddressesList.count > 0
             Component.onCompleted: derivedAddresses.pathSubFix = Qt.binding(function() { return pathSubFix})
         }
+
+        StatusButton {
+            visible: _internal.showEnterPinPassButton
+            text: derivedAddresses.selectedKeyUidMigratedToKeycard || userProfile.isKeycardUser?
+                      qsTr("Enter PIN") :
+                      qsTr("Enter password")
+            icon.name: derivedAddresses.enterPasswordIcon
+            highlighted: focus
+
+            Keys.onReturnPressed: _internal.runAction()
```

Comment:
How can you press Return on a button here? :)

Reply:
@caybro that's the consequence of copy/paste (I copied button from `AddAccountModal.qml`), that's why I said what I said in the description of this PR. :)